packages/angular-sdk-components/src/lib/_components/field/multiselect/multiselect.component.html

```diff
@@ -0,0 +1,36 @@
+<div [formGroup]="formGroup$">
+  <mat-form-field class="psdk-full-width" subscriptSizing="dynamic">
+    <!-- <div class="chip"> -->
```

Comment:
Can we get rid of this commented out div?

Reply:
sure , will do that

Reply:
done